code/game/objects/effects/spiders.dm

```diff
@@ -22,9 +22,9 @@
 
 /obj/effect/spider/attackby(obj/item/W, mob/user)
 	if(LAZYLEN(W.attack_verb))
-		visible_message(SPAN_DANGER("<B>\The [src] have been [pick(W.attack_verb)] with \the [W][(user ? "by [user]." : ".")]"))
+		visible_message(SPAN_DANGER("[src] has been [pick(W.attack_verb)] with \the [W][(user ? "by [user]." : ".")]"))
 	else
-		visible_message(SPAN_DANGER("<B>\The [src] have been attacked with \the [W][(user ? "by [user]." : ".")]"))
+		visible_message(SPAN_DANGER("[src] has been attacked with \the [W] [(user ? "by [user]." : ".")]"))
```

Comment:
Since the options are either `"by user."` or just `"."`, it would be better to have the space as part of the first option rather than before it. That way it won't be included in the message if it goes with the second one.
`"The web has been attacked with the thing ."` / `"The web has been attacked with the thing."`
```suggestion
		visible_message(SPAN_DANGER("[src] has been attacked with [W][(user ? " by [user]." : ".")]"))
```
(also may as well remove the other `\the` too while you're here)